aemedge/blocks/header/header.css

```diff
@@ -1,83 +1,205 @@
 /* header and nav layout */
-header {
-  height: 60px;
-  background: transparent;
-}
-
 header .nav-wrapper {
+  background-color: var(--background-color);
   width: 100%;
   z-index: 2;
   position: fixed;
-  background: transparent;
 }
 
 header nav {
-  display: grid;
-  grid-template:
-    'brand tools hamburger' var(--nav-height) 'sections sections sections' 1fr / auto 1fr auto;
-  align-items: center;
-  gap: 0 2em;
+  box-sizing: border-box;
   margin: auto;
-  height: 60px;
-  padding: 0 1rem;
+  width: 100vw;
+  position: absolute;
+  height: var(--nav-height);
   font-family: var(--body-font-family);
-  background: transparent;
+  overflow: hidden;
+  transition: height 0.5s ease-in-out;
+  background: linear-gradient(360deg, #292828fa 0%, #484848fa 100%);
 }
 
-header nav[aria-expanded="true"] {
-  grid-template:
-    'brand hamburger' var(--nav-height) 'sections sections' 1fr
-    'tools tools' var(--nav-height) / auto 1fr;
-  overflow-y: auto;
-  min-height: 500px;
-  background-color: var(--header-text-color);
-  color: var(--header-background-color);
+header nav[aria-expanded='true'] {
+  height: 100dvh;
 }
 
-header nav[aria-expanded="false"] {
-  background-color: var(--header-background-color);
-  color: var(--header-text-color);
+header nav .navprimary .nav-heading-title {
+  display: block;
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 18px;
+  color: #f5f5f6 !important;
 }
 
-header nav .nav-brand {
-  top: 0;
+header .profile .icon.icon-sign-in {
+  position: absolute;
+  top: 15px;
+  right: 60px;
+  width: 32px;
+  height: 32px;
 }
 
-header nav[aria-expanded="true"] .nav-brand {
-  width: calc(100% - 32px);
-  background-color: var(--header-text-color);
-  padding-top: 16px;
+header .icon.icon-logo-sling-rebrand {
+  margin-left: 16px;
+  margin-top: 10px;
 }
 
-header nav[aria-expanded="false"] .nav-brand {
-  top: unset;
+header .icon.icon-logo-sling-rebrand img {
+  height: 32px;
+  width: 55.77px;
 }
 
-header nav .nav-brand img {
-  width: 68px;
-  height: 26px;
-  vertical-align: center;
+header .section.signup {
+  display: none;
+}
+
+header nav .nav-sections ul {
+  list-style: none;
+  padding-left: 0;
+  font-size: var(--body-font-size-s);
 }
 
 header nav .nav-sections ul > li {
+  font-weight: 500;
+}
+
+header nav .nav-sections ul > li > ul {
+  margin-top: 0;
+}
+
+header nav .nav-sections ul > li > ul > li {
+  font-weight: 400;
+}
+
+header nav[aria-expanded='true'] .nav-drop ul {
+  display: flex;
+  width: 100vw;
+  position: fixed;
+  flex-direction: column;
+  -webkit-box-pack: justify;
+  height: calc(-60px + 100dvh);
+  transform: translateX(100%);
+  left: 0;
+  top: 140px;
+  margin-left: 27px;
+  transition: height 0.5s ease-in-out, transform 0.5s ease-in-out;
+}
+
+header nav[aria-expanded='true'] .navsecondary {
+  margin-top: 300px;
```

Comment:
Remove this margin, instead add this so that part of the nav is always in view at the bottom.
```
    position: absolute;
    bottom: 0;
```